### PR TITLE
Fix mobile menu jump & double border

### PR DESCRIPTION
**Pull request by MartijnCuppens**
Preview: https://deploy-preview-31975--twbs-bootstrap.netlify.app/docs/4.5/getting-started/introduction/

### CODE REVIEW

**Review by XhmikosR — APPROVED** [review]

**Comment by mdo** [comment]
This introduces some movement in the borders that doesn't complete until the animation does. Anyway around that?

File: site/_includes/docs-sidebar.html [hunk 1]
```diff
@@ -5,39 +5,41 @@
...
 </form>
 
-<nav class="collapse bd-links" id="bd-docs-nav" aria-label="Main navigation">
-  {%- assign page_slug = page.url | split: '/' | last -%}
-  {%- for group in site.data.nav -%}
-  {%- assign link = group.pages | first -%}
-  {%- assign link_slug = link.title | slugify -%}
-  {%- assign group_slug = group.title | slugify -%}
-  {%- assign active = nil -%}
+<div class="collapse d-lg-block" id="bd-docs-nav">
```
Comment (MartijnCuppens) [inline site/_includes/docs-sidebar.html:8]: Fix wrong breakpoint
```suggestion
<div class="collapse d-md-block" id="bd-docs-nav">
```

**Comment by MartijnCuppens** [comment]
> This introduces some movement in the borders that doesn't complete until the animation does. Anyway around that?

What movement? 😀

**Comment by mdo** [comment]
See how the border here doesn't extend the full width. This happens when the sidebar is collapsing, but is fixed when it's collapsed or fully opened.

Screen Shot 2020-10-26 at 1 26 18 PM

**Review by mdo — APPROVED** [review]

**Pull request merged**